hail_scripts/reference_data/config.py

```diff
@@ -43,6 +58,11 @@ def custom_gnomad_select_v3(ht):
                               0, ht.freq[ht.globals.freq_index_dict['XY-adj']].AC)
     return selects
 
+def custom_screen_select(ht):
```

Comment:
I made an attempt to do this enum mapping with a config-based approach.  

Something like:

```
'enum_definitions' : [{
  'dst_field_name': 'regionType_ids',
  'src_field_name': 'target',
  'lookup': SCREEN_REGION_TYPE_LOOKUP,
}]
```

might work?  And it would save us from having both `custom_select` and an `enum_definitions` field in the config.

Reply:
I'm gonna go ahead and implement this.   With fresh eyes I think this is cleaner.